torch/_inductor/cpp_builder.py

```diff
@@ -183,6 +183,10 @@ def _is_gcc(cpp_compiler: str) -> bool:
     # Since "clang++" ends with "g++", the regex match below would validate on it.
     if _is_clang(cpp_compiler):
```

Comment:
Isn't it already caught here?

Reply:
Thanks you @desertfire， 
I checked `git log` this PR is inqueue too much time, and https://github.com/pytorch/pytorch/pull/147775 was fixed this issue. I will close this PR as the issue was fixed.